changelog.md

```diff
@@ -23,6 +23,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `getDefinedAttributesByType` will now return nested attributes inside object, fix [this bug](https://github.com/ditrit/leto-modelizer-plugin-core/issues/203).
 
+- Add `arrangeComponentsPosition` method in `DefaultDrawer` and `DefaultPlugin`
```

Comment:
I dont see the addition of `arrangeComponentsPosition` method in `DefaultDrawer` and `DefaultPlugin`? The function already existed